### DISCUSSION TITLE
When an issue is closed, notify maintainers of repositories that refer to it

### DISCUSSION THREAD

**Discussion by kubukoz** [community/community#38202] [Projects and Issues] [answered]
Hello!

When I'm implementing a workaround for an issue, I'll often leave a link to the issue as a code comment. It would be amazing if, when the linked issue is closed, the repo maintainers got some sort of notification (maybe even an automatically created issue) saying that a linked issue has been resolved - this would be likely to allow removing such workarounds from the downstream repository.

**Comment by david-engelmann** [comment, marked as answer]
This feels like something that could be completed with a github action workflow. The trigger would be issue closed, you could configure the notification to yourself in a bunch of different ways. Send an email, or if you'd like to keep it in GitHub, create a new issue automatically.

**Reply by kubukoz** [reply]
As I understand, this would require the action to be added to the upstream?

**Reply by kubukoz** [reply]
ah, I think I get it - the action could run in my repo periodically, and when it sees a link to a closed issue it could do the actual reporting. I think that'd be good enough, it doesn't need to be instantaneous.

**Comment by clorteau** [comment]
Solution in comment but issue still open (because  issue still open?)

**Reply by kubukoz** [reply]
It's a discussion so I don't think it can be closed per se. There's a solution that's been marked as an answer so I guess that's it
